department-of-veterans-affairs/caseflow
# Omar/caseflow 2267 fix schedule veteran form

Resolves [CASEFLOW-1752](https://vajira.max.gov/browse/CASEFLOW-1752)
Resolves [CASEFLOW-2267](https://vajira.max.gov/browse/CASEFLOW-2267)

### Description
It's possible to have a hearing day selected in the "Hearing Date" dropdown, but have no time selection UI appear on the screen. This makes it impossible to select a time for the hearing.

### Acceptance Criteria
- [ ] Make sure that the 2 bugs described in the tickets are not occurring anymore.

### Testing Plan
1. Follow instructions on the tickets to reproduce.
2. Make sure it breaks.
3. Checkout the branch ` git checkout omar/CASEFLOW-2267-fix-schedule-veteran-form`
4. Try to reproduce again.
5. Apart from the steps on the tickets makes sure that the `HearingDateDropdown` works when postponing a Hearing.
